test/blackbox-tests/test-cases/pkg/ocamlformat-dev-tool.t

```diff
@@ -33,11 +33,11 @@ Make a fake ocamlformat:
   $ tar -czf ocamlformat-0.26.3.tar.gz ocamlformat
   $ rm -rf ocamlformat
 
-Add the tar file for the fake curl to copy it:
+Add the `tar` file for the fake curl to copy it:
   $ echo ocamlformat-0.26.2.tar.gz > fake-curls
   $ PORT=1
 
-Make a package for the fake ocamlformat library:
+Make a package for the fake `ocamlformat` library:
```

Comment:
double checking that "`ocamlformat` library" should be in monospace, or should it be "OCamlFormat library"?

Reply:
These aren't markdown files, so adding backticks is discouraged.